default.nix

```diff
@@ -45,6 +45,10 @@ let
     settings.formatter.shfmt.options = [ "--space-redirects" ];
   };
 
+  # The resulting package is built to always use this Nix version, such that the result is reproducible
+  # TODO: Switch this to pkgs.nixVersions.minimum, because that's what Nixpkgs CI should use
```

Comment:
I'm interested in the reasoning behind this "should". My experience is that it's better if CI uses the most recent version of its dependencies, since that usually means that fixes can occur at the right level. Using the minimum version, in my experience, leads to hard-to-remove workarounds for old bugs, which don't contribute to a better product. And the process of upgrading the minimum version has a nasty habit of finding odd new local minima, which then need to be worked around again.

The current Nixpkgs way of doing things -- having a stable version, which moves forward on a ~six monthly cadence in normal times -- is my preferred CI set up.

Reply:
Oh yeah, we totally shouldn't have to use the minimum version here. The minimum version should only be used for the part of CI that checks that everything evaluates, which is ofborg (though that [actually use the minimum version](https://github.com/NixOS/ofborg/issues/612)!).

Removed the TODO now :)